frontend/src/metabase/query_builder/components/VisualizationError/tests/common.unit.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import { createMockCard, createMockDatabase } from "metabase-types/api/mocks";
+import { screen } from "__support__/ui";
+import { setup } from "./setup";
+
+describe("VisualizationError (OSS)", () => {
+  describe("SQL databases", () => {
+    const database = createMockDatabase({
+      engine: "postgres",
+    });
+
+    it("should show a help link when `show-metabase-links: true`", () => {
+      const card = createMockCard({
+        dataset_query: {
+          database: database.id,
+          type: "native",
+          native: {
+            query: "SELECT * FROM ORDERS",
+          },
+        },
+      });
+      setup({ database, card, showMetabaseLinks: true });
+
+      expect(
+        screen.getByText("Learn how to debug SQL errors"),
+      ).toBeInTheDocument();
+    });
+
+    it("should show a help link when `show-metabase-links: false`", () => {
+      const card = createMockCard({
+        dataset_query: {
+          database: database.id,
+          type: "native",
+          native: {
+            query: "SELECT * FROM ORDERS",
+          },
+        },
+      });
+      setup({ database, card, showMetabaseLinks: false });
+
+      expect(
+        screen.getByText("Learn how to debug SQL errors"),
+      ).toBeInTheDocument();
+    });
+  });
+
+  describe("NoSQL databases", () => {
+    const database = createMockDatabase({
+      engine: "mongo",
+    });
+
+    it("should show a help link", () => {
```

Comment:
Took me a while to understand why the describe was "should show" but the test was "not.toBeInTheDocument"
```suggestion
    it("should not show a help link about SQL queries for noSQL", () => {
```
(or some other copy)
Same for the other 2 files

Reply:
Fixed in f0c6e32f0577a0a69f75616bdbb7cc7d66ab4e38